src/components/app.scss

```diff
@@ -1,3 +1,5 @@
+$base-font-family: sans-serif !default;
```

Comment:
Can you remove the !default and move this after the normalize import?

Reply:
Hi,

I have tried to put it AFTER @import initially, but the font change did not kick in.

So I googled a bit, and tried it similar to what this stackoverflow post said (https://stackoverflow.com/questions/19793482/how-can-i-set-basefontfamily-for-bootstrap) by putting BEFORE it, and it worked.